test/files/pos/t11726/S_2.scala

```diff
@@ -0,0 +1,5 @@
+import jli._
+
+object Test {
+  val x = new SD_1.Q
+}
```

Comment:
Could also add a test for runtime reflection. I can't provoke a failure at the moment, so it would just defend the status quo.

```
scala> typeOf[jli.SD_1].companion.member(TypeName("Q")).info.baseType(symbolOf[jli.CS_1[_, _]])
res13: reflect.runtime.universe.Type = jli.CS_1[String,jli.SD_1]
```

Reply:
.. and it would definitely be worth testing this in joint compilation. (Bugs in this area sometime need to be fixed in `JavaParsers`, `JavaMirrors` and `ClassfileParser`)

Reply:
Done. That rabbit hole is deep.